galaxytool-import/action.yml

```diff
@@ -0,0 +1,35 @@
+name: "import galaxy tool"
+description: "galaxy tool bulk import"
+
+inputs:
+  repo-user:
+    description: "User that commits modifications"
+    required: true
+  repo-token:
+    description: "GitHub token to commit modifications"
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: "3.7"
+
+    - name: Install dependencies
+      run: pip install -r ${{ github.action_path }}/requirements.txt
+      shell: bash
+
+    - name: import galaxy tool using the API
+      run: |
+        python ${{ github.action_path }}/galaxytool-import.py
+      shell: bash
+
+    - name: reformat json to a predictable format with jq and sponge
+      run: |
+        sudo apt-get update -y
+        sudo apt-get install moreutils
+        FILES='data/*/*.galaxy.json'
```

Comment:
Because json files are both in imports/galaxy/*.galaxy.json and data/*/*.galaxy.json, I suggest modifying this mask to */*/*.galaxy.json.

Reply:
Or doing both `data/*/*.galaxy.json` and `imports/galaxy/*.galaxy.json`. your choice!